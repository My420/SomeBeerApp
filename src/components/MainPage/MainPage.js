import React from 'react';
import Main from '../Main/Main';
import SlideShow from '../SlideShow/SlideShow';
import Container from '../Container/Container';
import styles from './MainPage.module.scss';
import Logo from '../Logo/Logo';
import SloganShakespeare from '../SloganGallery/SloganShakespeare/SloganShakespeare';
import SloganJohnnyDepp from '../SloganGallery/SloganJohnnyDepp/SloganJohnnyDepp';
import SloganChurchhill from '../SloganGallery/SloganChurchhill/SloganChurchhill';
import SloganLuther from '../SloganGallery/SloganLuther/SloganLuther';
import PopularGoods from '../PopularGoods/PopularGoods';

const MainPage = () => {
  return (
    <Main className={styles.main}>
      <Container className={styles.container}>
        <Logo />
        <SlideShow>
          <SloganShakespeare />
          <SloganJohnnyDepp />
          <SloganChurchhill />
          <SloganLuther />
        </SlideShow>
        <PopularGoods />
      </Container>
    </Main>
  );
};

export default MainPage;
