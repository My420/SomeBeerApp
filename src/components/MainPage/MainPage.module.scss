@import '../../sass/variables.scss';

.container {
  width: $PC-container-width;

  margin: 0 auto;
  padding: 0 20px;
}

@media (max-width: $PC-max-width) {
  .container {
    width: $MPC-container-width;
  }
}

@media (max-width: $MPC-max-width) {
  .main {
    padding-top: 60px;
  }

  .container {
    box-sizing: border-box;
    width: 100%;
  }
}
