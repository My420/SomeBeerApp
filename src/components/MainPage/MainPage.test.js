import React from 'react';
import { shallow } from 'enzyme';
import MainPage from './MainPage';

describe('Test <MainPage /> component', () => {
  const wrapper = shallow(<MainPage />);

  test('should correct render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
