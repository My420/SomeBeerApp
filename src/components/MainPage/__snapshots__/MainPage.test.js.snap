// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <MainPage /> component should correct render component 1`] = `
<Main
  className="main"
>
  <Container
    className="container"
  >
    <Logo />
    <SlideShow>
      <SloganShakespeare />
      <SloganJohnnyDepp />
      <SloganChurchhill />
      <SloganLuther />
    </SlideShow>
    <PopularGoods />
  </Container>
</Main>
`;
